src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
```

Comment:
The variable name 'stylesArray' could be improved for better readability. Consider naming it to reflect that it contains style declarations, such as 'styleDeclarations'.